opencog/query/PatternMatchCallback.h

```diff
@@ -238,6 +238,16 @@ class PatternMatchCallback
 		                       const std::map<Handle, Handle> &term_soln) = 0;
 
 		/**
+		 * Called when we are comparing unordered links. It would be useful
+		 * if we want to skip this grounding and move on to the next right away
+		 * without having to compare all of its possible permutations.
+		 */
```

Comment:
I don't know if I understood this correctly. For example if there are two groundings G1 and G2, both are unordered links, and G2 is the one we want but not G1. Currently if G1 reaches the grounding first, it seems that we have to go through all permutations of G1 before G2 reaching grounding?